tasks/copy_to_archive/README.md

```diff
@@ -1,15 +1,15 @@
-[![Known Vulnerabilities](https://snyk.io/test/github/nasa/cumulus-orca/badge.svg?targetFile=tasks/copy_to_glacier/requirements.txt)](https://snyk.io/test/github/nasa/cumulus-orca?targetFile=tasks/copy_to_glacier/requirements.txt)
+[![Known Vulnerabilities](https://snyk.io/test/github/nasa/cumulus-orca/badge.svg?targetFile=tasks/copy_to_archive/requirements.txt)](https://snyk.io/test/github/nasa/cumulus-orca?targetFile=tasks/copy_to_archive/requirements.txt)
 
 Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/development-guide/code/contrib-code-intro) for information on environment setup and testing.
 
 ## Description
 
-The `copy_to_glacier` module is meant to be deployed as a lambda function that takes a Cumulus message, extracts a list of files, and copies those files from their current storage location into a staging/glacier ORCA S3 bucket. It also sends additional metadata attributes to metadata SQS queue needed for Cumulus reconciliation.
+The `copy_to_archive` module is meant to be deployed as a lambda function that takes a Cumulus message, extracts a list of files, and copies those files from their current storage location into an ORCA archive bucket. It also sends additional metadata attributes to metadata SQS queue needed for Cumulus reconciliation.
```

Comment:
Since the forward facing name `copy_to_orca` is different then the name in the repository we should maybe denote in the README that the end user will be making the config/environment changes on a lambda named `PREFIX_copy_to_orca` in there step functions and usage.